pom.xml

```diff
@@ -153,24 +153,25 @@
             <version>${jackson.version}</version>
         </dependency>
 
+        <!-- YAML config parsing dependencies -->
```

Comment:
Fwiw these comments tend to go stale pretty quickly as the project grows. For example - Jackson `core` and `databind` are now general serialization dependencies used for both json and yaml parsing. What will happen over time is folks will use a dependency we've already pulled in (ie, `commons-text`) and not realize there's an associated POM comment that is no longer valid.

Reply:
I agree this is true with code comments but for the dependencies it seemed to work well atleast so far. I am flexible either ways but these 2 dependencies are related to yaml, so I thought I call them out explicitly.